perf/benchmark/runner/runner.py

```diff
@@ -338,7 +338,9 @@ def run(self, headers, conn, qps, size, duration):
             # See the comment above, we restrict execution to a single nighthawk worker for
             # now to avoid noise.
             workers = 1
-            jitter_uniform = float(0.1 * 1 / qps)
+            jitter_uniform = 0
```

Comment:
@oschaaf if `jitter_uniform` is 0s, it has same effect as no-jitter right? Or we should not pass this param when there is no-jitter?

Reply:
Passing `0s` will do no harm, and work as intended.

Reply:
Sorry, I thought let's add a quick test in NH to pin this case, and I should take that back. 0 gets rejected based on the options proto spec [1], which has:

```proto3
  google.protobuf.Duration jitter_uniform = 25 [(validate.rules).duration.gte.nanos = 1];
```

We should either not pass the parameter in that case, or modify Nighthawk's options proto to accept it, in which case it will just work (I verified that part, which led me to think this wouldn't be a problem in the first case). 

[1] https://github.com/envoyproxy/nighthawk/blob/master/api/client/options.proto#L183

Reply:
Yeah, I can do on my side by not passing that param. Not sure about NH side, depends on you if you'd prefer to update that proto or not.

Reply:
Filed https://github.com/envoyproxy/nighthawk/issues/513 to track this in the NH side as well in case others run into it.